### DISCUSSION TITLE
Import works, export doesn't include textures/materials

### DISCUSSION THREAD

**Discussion by akutterxressential** [atteneder/glTFast#482] [General]
Hi folks,

when importing the attached GLB Unity 2021.2.16f does that properly and I can see the materials (check the image below)

[GLB Export.zip](https://github.com/atteneder/glTFast/files/9659624/GLB.Export.zip)
![After import](https://user-images.githubusercontent.com/69792737/192635721-1a148a7e-14cf-4df6-bc90-a7b55423b5a9.JPG)

But after exporting the whole scene the materials are missing, you can also find this in the ZIP attachment. When I add a cube and assign a newly created HDRP lit material to this room, the whole room is still white and the cube has the right texture.  Why is that?

**Comment by atteneder** [comment]
Import/export round-trip does not work yet. It's missing either

- Import to HDRP Lit
- Export from glTF shaders

See #385, #258, #351, #347
